tests/test_initialization.py

```diff
@@ -1,24 +1,40 @@
 from textwrap import dedent
 
+import pytest
 
-def test_django_setup_order_and_uniqueness(django_testdir, monkeypatch):
+
+@pytest.mark.parametrize("conftest_mark", ("tryfirst", "trylast"))
+def test_django_setup_order_and_uniqueness(conftest_mark, django_testdir, monkeypatch):
     """
     The django.setup() function shall not be called multiple times by
     pytest-django, since it resets logging conf each time.
     """
     django_testdir.makeconftest(
         """
+        import pytest
         import django.apps
-        assert django.apps.apps.ready
-        from tpkg.app.models import Item
```

Comment:
That's probably a dealbreaker: not being able to import models in your conftest at the toplevel anymore?!